installers/rpm-code/codes/srw.sh

```diff
@@ -33,43 +33,51 @@ srw_main() {
 
 srw_python_install() {
     install_pip_install primme
-    cd SRW/cpp/py
-    make python
-    cd ../..
-    srw_python_patch_srwlib
-    srw_python_patch_uti_math_eigen
-    codes_python_lib_copy env/work/srw_python/{{srwl,uti}*.py,srwlpy*.so}
-    find . -name srwlpy\*.so -exec rm {} \;
+    install_pip_install srwpy==4.0.0.b1
+    srw_srwpy_backwards_compatible
 }
 
-srw_python_patch_srwlib() {
-    patch --quiet env/work/srw_python/srwlib.py <<'EOF'
-diff --git a/env/work/srw_python/srwlib.py b/env/work/srw_python/srwlib.py
-index 9e052df..0ab17fb 100644
---- a/env/work/srw_python/srwlib.py
-+++ b/env/work/srw_python/srwlib.py
-@@ -10267,6 +10267,8 @@ def srwl_wfr_emit_prop_multi_e(_e_beam, _mag, _mesh, _sr_meth, _sr_rel_prec, _n_
-                         #srwl_uti_save_intens_ascii(resStokes2.to_deg_coh(), meshRes2, fpdc2, _n_stokes = 1, _arLabels = resLabelsToSaveDC, _arUnits = resUnitsToSaveDC, _mutual = doMutual, _cmplx = 0) #OC12072019 # Deg. of Coh. Cut vs Y
-                         #srwl_uti_save_intens_ascii(resStokes2.to_deg_coh(), meshRes2, fpdc2, _n_stokes = 1, _arLabels = resLabelsToSaveDC, _arUnits = resUnitsToSaveDC, _mutual = 2, _cmplx = 0) #OC16072019 # Deg. of Coh. Cut vs Y
-                         srwl_uti_save_intens(resStokes2.to_deg_coh(), meshRes2, fpdc2, _n_stokes = 1, _arLabels = resLabelsToSaveDC, _arUnits = resUnitsToSaveDC, _mutual = 2, _cmplx = 0, _form = _file_form) #OC17072021 # Deg. of Coh. Cut vs Y
-+                elif _char == 61 and i + 1 < nRecv:
```

Comment:
This patch is important so you need to look at the code in srwlib.py in the new release to see if we need to patch it.

Reply:
The existing patch works on the new file. 994d56ec4eb7008c310ccbcf15d10cae03f86f38